.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -17,11 +26,11 @@ Check documentation: https://checkstyle.org/config_xxxxxx.html#NameOfAffectedChe
 
 ---------------
 
-Describe what you expect in detail.
+**Describe what you expect in detail.**
 
 --------------
 
 Still not clear ???
 see example - https://checkstyle.org/report_issue.html#How_to_report_a_bug.3F
 
-ATTENTION: FAILURE TO FOLLOW THE ABOVE TEMPLATE WILL RESULT IN THE ISSUE BEING CLOSED.
+ATTENTION: FAILURE TO FOLLOW THE ABOVE TEMPLATE MAY RESULT IN THE ISSUE BEING CLOSED.
```

Comment:
Why was this wording changed?

Reply:
I've seen many issues which did not stick to the prescribed format being approved, as they were critical. Hence, I felt it would be better to have a bit less assertive tone here. If you don't feel this to be okay, I'll be more than happy to revert it.

Reply:
please revert this, unfortunately it is critical, only maintainers can create issue that is not following format. Unfortunately there are a exceptions from this rule, but it is decision of maintainers to keep bug report or not. 
This line is for bug report, it should follow format in 99% cases, and we are closing issues if author is not willing to do this.

Reply:
done